src/debug.c

```diff
@@ -1042,6 +1051,9 @@ NULL
             addReplyErrorFormat(c, "Unknown direction %s", (char*) c->argv[3]->ptr);
         }
         addReply(c,shared.ok);
+    } else if(!strcasecmp(c->argv[1]->ptr,"get-threads-stacktraces") && c->argc == 2) {
```

Comment:
a "get threads stacktrace" feature is a nice thing to have in redis (not for testing, but for runtime troubleshooting).
but, if we can, we better not have it as a command, since redis might be unresponsive, which is the reason why we want to extract a stack trace.

i think we should re-use the SIGALRM mechanism used by `watchdog-period`, and just enable it on startup regadless of the config.
then whenever we want a stacktrace, we just send the signal from command line.

Reply:
added this to the following issue 
https://redislabs.atlassian.net/browse/MOD-5536?atlOrigin=eyJpIjoiYjcwZTRiODA3M2MwNDJlZWFkNTdjNTc2YTczZjNjZmMiLCJwIjoiaiJ9